apps/frontend/src/lib/storage/localStorage/utils.ts

```diff
@@ -8,6 +8,7 @@ export const saveToLocalStorage = (key: string, value: string): void => {
   localStorage.setItem(key, value);
 };
 
+// uncaughtException: ReferenceError: localStorage is not defined
 export const getFromLocalStorage = (key: string): string | null => {
```

Comment:
Been running into this runtime issue where the browser is confusing the native window LocalStorage with the node LocalStorage.

Reply:
I don't see what change I introduced that could create this behavior. I imported ws and ws types, nothing to do with LocalStorage.

Reply:
Even when I use `window.localStorage.getItem(key);` it complains with the same error even though I can follow the reference correctly, very weird.